packages/core/admin/admin/src/components/AutoReloadOverlayBlockerProvider/Blocker.js

```diff
@@ -43,38 +44,48 @@ const Blocker = ({ displayedIcon, description, title, elapsed, isOpen }) => {
   if (isOpen) {
     return ReactDOM.createPortal(
       <Overlay>
-        <Box>
-          <Row>
+        <Content size={6}>
+          <Stack size={2}>
+            <Row justifyContent="center">
+              <H1>{formatMessage(title)}</H1>
+            </Row>
+            <Row justifyContent="center">
+              <Typography as="h2" textColor="neutral600" fontSize={4} fontWeight="regular">
+                {formatMessage(description)}
+              </Typography>
+            </Row>
+          </Stack>
+          <Row justifyContent="center">
             {displayedIcon === 'reload' && (
-              <Box paddingRight={3} style={{ alignSelf: 'baseline' }}>
-                <LoaderReload width="4rem" height="4rem" />
-              </Box>
+              <IconBox padding={6} background="primary100" borderColor="primary200">
+                <LoaderReload width={pxToRem(37)} height={pxToRem(37)} />
```

Comment:
37? Prime number? 😬  Could you switch to 36? It's closer to the spacing scale of the DS